pkg/test/certificates/foo.crt

```diff
@@ -0,0 +1,35 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Theere is already a `pkg/test/assets/assets.go` file with a certificate inside. Couldn't we unify both things in only one certificate?

Reply:
The files are not longer used, as they were required from different tests and the path for accessing them was relative to the test's directory. Therefore the certs were embedded in a go source file in the pkg/test/asset package.